doc/xrcvd/transceiver-monitor-hld.md

```diff
@@ -91,17 +90,15 @@ New Transceiver info table and transceiver DOM sensor table will be added to sta
         key                          = TRANSCEIVER_STATUS|ifname     ; Error information for SFP on port
         ; field                      = value
         status                       = 1*255VCHAR                    ; code of the SFP status (plug in, plug out)
-        error                        = 1*255VCHAR                    ; SFP error (N/A or a string like error1 | error2 )
-
+        error                        = 1*255VCHAR                    ; SFP error (N/A or a string consisting of error descriptions joined by "|", like "error1 | error2" )
 
 ### 1.2 Accessing EEPROM from platform container ###
 
-Transceiver EEPROM information can be accessed via reading sysfs files(e.g. `/sys/bus/i2c/devices/2-0048/hwmon/hwmon4/qsfp9_eeprom`) or other ways, this is upon vendor's own implementation. 
-
-Transceiver EEPROM accessing can be achieved by legacy sfputil.py plugin or new platform API, Xcvrd supports both of these two methods. If platform API not yet implemented on some vendor's device, it will automatically fall back on sfputil.py plugin.
+Transceiver EEPROM information can be accessed via reading sysfs files(e.g. `/sys/bus/i2c/devices/2-0048/hwmon/hwmon4/qsfp9_eeprom`) or other ways, this is upon vendor's own implementation.
 
+Transceiver EEPROM accessing can be achieved by legacy sfputil.py plugin or new platform API. Xcvrd supports both methods. If platform API is not yet implemented on some vendor's device, it will automatically fall back on sfputil.py plugin.

```

Comment:
I think we can remove any references to the legacy plugins. They have been deprecated for a couple years now, and I am starting to remove support little by little. Feel free to remove references throughout the document.